frontend/src/utils/ropeApi.ts

```diff
@@ -0,0 +1,59 @@
+export interface User {
+  email: string
+  is_admin: boolean
+  is_manager: boolean
```

Comment:
:thinking: -- I would expect that this interface would use camelCase, and internal to `ropeApi` we convert as needed when invoking / parsing responses from the API. Otherwise, right now we still end up leaking `snake_case` in the `Page.tsx` code which I think we'd like to avoid :man_shrugging: .